addon/system/store.js

```diff
@@ -499,12 +500,16 @@ Store = Service.extend({
     @param {Object} options
     @return {Promise} promise
   */
-  findRecord(modelName, id, options) {
+  findRecord(modelName, id, options = {}) {
     assert('Passing classes to store methods has been removed. Please pass a dasherized string instead of '+ Ember.inspect(modelName), typeof modelName === 'string');
     assert(badIdFormatAssertion, (typeof id === 'string' && id.length > 0) || (typeof id === 'number' && !isNaN(id)));
 
-    var internalModel = this._internalModelForId(modelName, id);
-    options = options || {};
+    const internalModel = this._internalModelForId(modelName, id);
+    const { include } = options;
+
+    if (include) {
+      options.adapterOptions = merge(options.adapterOptions || {}, { include });
```

Comment:
@igorT points out that we may not want to use `adapterOptions` since that is a namespace for the users to put whatever they'd like. Instead we could put possibly put `include` on the snapshot. **This is still to be decided...**